docs/readme.md

```diff
@@ -260,6 +267,34 @@ CalevBehaviors.withCalendarEvent(calEvent)(
       same
   }
 )
+```
+#### Testing
+
+See [CalevBehaviorsTest](https://github.com/eikek/calev/blob/master/modules/akka/src/test/scala/com/github/eikek/calev/akka/dsl/CalevBehaviorsTest.scala)
+
+#### Akka Scheduler 
+
+Schedule the sending of a message to the given target Actor at the time of 
+the upcoming event according to the given calendar event definition.
 
+```scala mdoc
+def behavior(tickReceiver: ActorRef[Tick]): Behavior[Message] = 
+  setup { actorCtx =>
+    actorCtx.scheduleOnceWithCalendarEvent(calEvent, tickReceiver, Tick)
+    same
+  }
 ```
-More examples to come...
+
+Schedule the running of a ```Runnable``` at the time of the upcoming 
+event according to the given calendar event definition.
+
+```scala mdoc
+implicit val system: ActorSystem[_] = ActorSystem(empty, "my-system")
```

Comment:
Regarding readme; can it be that here an actor system is started that has non-daemon threads running? This could prevent the jvm from quitting.

Reply:
If I add a `system.terminate()` at the end it seems to work…?

Reply:
Yes,  I added system.terminate().